pep-0008.txt

```diff
@@ -850,6 +850,30 @@ names.
 In some fonts, these characters are indistinguishable from the
 numerals one and zero.  When tempted to use 'l', use 'L' instead.
 
+ASCII Compatibility
+~~~~~~~~~~~~~~~~~~~
+
+While support for Unicode identifiers has been added to allow all
+Python developers to use variable names in their native languages,
+the 7-bit ASCII text encoding remains the lowest common denominator
+for text entry when developing Python programs.
+
+Accordingly, all identifiers in the Python standard library must be
+ASCII-only identifiers.
+
+In addition, string literals and comments used in the reference
+implementation should also be ASCII compatible. The only exceptions
+are (a) test cases testing the non-ASCII features, and (b) names of
+authors. Authors whose names are not based on the Latin alphabet
+MUST provide a Latin transliteration of their names. 
+
+Readability as English pseudo-code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Standard library Identifiers should be English words wherever
+feasible. However, abbreviations and technical terms which aren't
+English may also be used when appropriate.
```

Comment:
Your rewrite changes the meaning (PEP 3131 merely states that non-English things *are* used).